packages/react/ds-app-launchpad/src/ui/index.ts

```diff
@@ -1 +1,2 @@
 export { Button, type ButtonProps } from "./Button/index.js";
+export { EditableBlock, type EditableBlockProps } from "./EditableBlock/index.js"
```

Comment:
this import is not importing your editable block from `./shared` and causing failing builds